core/src/main/java/org/dst/core/DstAbstractMap.java

```diff
@@ -0,0 +1,13 @@
+package org.dst.core;
+
+public interface DstAbstractMap<K, V> {
```

Comment:
Abstract may be a abstract class?

How about DstStorage?

Reply:
We have a interface named `KVStore`, if we named this as `DstStorage`, I am afraid to confuse them.